packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -307,6 +354,16 @@ export function signInActor({ services }: SignInMachineOptions) {
           const { confirmation_code: challengeResponse } = formValues;
           return services.handleConfirmSignIn({ challengeResponse });
         },
+        handleSetupEmail({ formValues }) {
+          return services.handleConfirmSignIn({
+            challengeResponse: formValues.email,
+          });
+        },
+        handleSelectMfaType({ formValues }) {
+          return services.handleConfirmSignIn({
+            challengeResponse: formValues.mfa_type,
+          });
+        },
```

Comment:
Any reason not to simplify so only `services.confirmSignIn` is used for all `challengeResponse` handling?